### PR TITLE
Release notes for 2023-09-17

### PR DESCRIPTION
**Pull request by AllenWen-at-Azure**

### CODE REVIEW

**Review by palma21 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+## Release 2023-09-17
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+* Behavioral changes
+  * After you set the [node OS auto-upgrade channel](https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-image#using-node-os-auto-upgrade) to "None", AKS doesn't automatically reimage nodes in your node pools. But when you set the node OS auto-upgrade channel to "Unmanaged", AKS will reimage all nodes in your node pools.
```
Comment (CocoWang-wql) [inline CHANGELOG.md:12]: https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-image#using-node-os-auto-upgrade  -> remove  en-us

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+* Features
+  * [HTTP Proxy](https://learn.microsoft.com/en-us/azure/aks/http-proxy#updating-proxy-configurations) can now be updated post clusters creation.
```
Comment (CocoWang-wql) [inline CHANGELOG.md:15]: https://learn.microsoft.com/en-us/azure/aks/http-proxy#updating-proxy-configurations  -> remove en-us

File: CHANGELOG.md (hunk 1 above)
Comment (AllenWen-at-Azure) [reply CHANGELOG.md]: fixed it!

File: CHANGELOG.md (hunk 2 above)
Comment (AllenWen-at-Azure) [reply CHANGELOG.md]: fixed it!

**Review by CocoWang-wql — APPROVED** [review]

**Pull request merged**